### PR TITLE
Add trusted apps for cluster-proxy components.

### PR DESCRIPTION
**Pull request by xuezhaojun**

### CODE REVIEW

**Comment by xuezhaojun** [comment]
/assign @Kyl-Bempah 

Looking for approving! thanks

File: core-services/prow/02_config/stolostron/cluster-proxy-addon/_pluginconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+plugins:
```
Comment (zhujian7) [inline core-services/prow/02_config/stolostron/cluster-proxy-addon/_pluginconfig.yaml:1]: Not sure if an `OWNERS` file is required?

Comment (xuezhaojun) [reply core-services/prow/02_config/stolostron/cluster-proxy-addon/_pluginconfig.yaml:1]: I thought OWNERS are auto-genrated by prow, I will follow up, if it's not created, I will create it.

**Pull request merged**